### DISCUSSION TITLE
HowTo: Adding additional dependencies

### DISCUSSION THREAD

**Discussion by SeeJayDee** [rust-cross/rust-musl-cross#105] [Q&A]
Hi, I'm building for RPi3/aarch64 target on Windows amd64.

I need to add dependencies:
`libasound2-dev`
`libssl-dev`
`libpulse-dev`
`libdbus-1-dev`

...but unsure how to do this correctly.
Presumably adding those packages into the `RUN ... apt-get` line of the Dockerfile will only install the amd64 versions.

I usually do OK just looking through the source and tweaking it to my needs but in this case I'm stumped.
Any help would be appreciated! :)

(Also happy to write up a guide covering what steps I took, if you want to add such a thing to the repo later)

**Comment by messense** [comment]
Something like this

```bash
dpkg --add-architecture arm64
apt-get update
apt-get --assume-yes install libssl-dev:arm64
```

See also https://wiki.debian.org/Multiarch/HOWTO
